security/https/src/test/resources/windows.properties

```diff
@@ -1,9 +1,2 @@
 quarkus.http.ssl.certificate.key-store-file=META-INF\\resources\\server-keystore.pkcs12
-quarkus.http.ssl.certificate.key-store-password=server-password
-quarkus.http.ssl.certificate.trust-store-file=META-INF\\resources\\server-truststore.pkcs12
-quarkus.http.ssl.certificate.trust-store-password=server-password
-quarkus.http.ssl.client-auth=request
-quarkus.ssl.native=true
-quarkus.http.auth.policy.users-only.roles-allowed=user
-quarkus.http.auth.permission.secured-urls.paths=/secured/*
-quarkus.http.auth.permission.secured-urls.policy=users-only
```

Comment:
This is no longer needed (it's already defined in the default `application.properties`).